Makefile

```diff
@@ -34,7 +34,7 @@ DEB_TARGET       = $(BUILD_DIR)/$(PROJECT_BIN)_$(VERSION_STR).deb
 
 $(PROJECTS): $(GOBIN)
 	$(eval proj := $@)
-	go build -i -ldflags "$(LDFLAGS)" -o $(BIN_TARGET) $(PROJECT_PKG)
+	go build -mod=vendor -i -ldflags "$(LDFLAGS)" -o $(BIN_TARGET) $(PROJECT_PKG)
```

Comment:
This is to ensure we're building with the dependencies in the `vendor/` directory